### ISSUE TITLE
Values of input bindings cannot be obtained

### ISSUE THREAD

**Issue by RootG** [MovingBlocks/Terasology#2662]
I am trying to show values of toolbar bindings on toolbars to improve usability. But I could not find a way to do it without using Config class, which is not available in Core module.

### This is what I am trying to do.
![inventorylast](https://cloud.githubusercontent.com/assets/17679464/21119455/50abc48e-c0cb-11e6-8e4d-ba294a702d6d.png)

I created a new API annotated class in Engine module to obtain values of input bindings. I think there should be a utility class or something that allow modules to access Config class.
##### This is the class I wrote.
```
package org.terasology.config;

import org.terasology.engine.SimpleUri;
import org.terasology.input.Input;
import org.terasology.module.sandbox.API;
import org.terasology.registry.CoreRegistry;

import java.util.List;

@API
public final class InputConfigUtils {

    private InputConfigUtils() {
    }

    public static Input getBindByUri(SimpleUri uri) {
        Config config = CoreRegistry.get(Config.class);
        BindsConfig bindConfig = config.getInput().getBinds();
        List<Input> binds = bindConfig.getBinds(uri);
        return binds.get(0);
    }
}
```

**Comment by Cervator** [comment]
Yeah config at the module level is still needing some work for sure :-)

Curiously, is the idea here that you want to be able to render a *custom* binding (in case the player changed from a number to some arbitrary key) rather than just enumerate the toolbar slots? Since really just hard coding a display of 1, 2, 3, 4.. on each slot would help in probably 99% of cases, and wouldn't need to check config :-)

That way we'd just be better clarifying which slot is which, if somebody likes having the numbers handy. If rebinding probably the user would assign a different row of keys, or just pick something unique for one or two slots?

If you think a player would commonly want to rebind all the toolbar slots to custom irregular keys then I can see the more sophisticated approach being worth it, but if not it seems somewhat overkill?

**Comment by RootG** [comment]
Yeah, you are right, it's overkill. I never thought about it that way. I was just trying to get familiar with the source code. I even implemented much more sophisticated approach by enabling/disabling this feature with a console command. :smile:
